daphne/src/roles.rs

```diff
@@ -322,7 +357,7 @@ pub trait DapLeader<S>: DapAuthorizedSender<S> + DapAggregator<S> {
             self,
             task_id,
             task_config,
-            "/aggregate",
+            "aggregate",
```

Comment:
This appears to be an accidental regression (here and below)
```suggestion
            "/aggregate",
```

Reply:
This is a side-effect of using `url::Url` `join()`.